packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -334,6 +345,40 @@ abstract contract Sphinx {
         return sphinxUtils.getGnosisSafeProxyAddress(address(this));
     }
 
+    /**
+     * @notice Utility function that allows the user to transfer funds to their safe at
+     * the beginning of a deployment. A utility function is required for this (vs just
+     * using _to.call{value: _amount} or transfer()) because this features involves the
+     * DevOps platform transferring funds directly to the safe from an EOA as part of the
+     * deployment. _to.call{value: _amount} or transfer() can be used to transfer value
+     * from the Safe to other addresses.
+     */
+    function fundSafe(uint _value) public {
+        if (fundsRequestedForSafe > 0) {
+            revert(
+                "Sphinx: Safe funds transfer already requested on this network. You're limited to one funding transfer to your safe per network."
+            );
+        } else {
+            sphinxUtils.checkValidSafeFundingRequest(_value);
+            safeStartingBalance = safeAddress().balance;
+            fundsRequestedForSafe = _value;
+
+            // Update the balance of the safe to equal to Safe's current balance + the
+            // value that we will transfer to the Safe during the deployment.
+            vm.deal(safeAddress(), safeStartingBalance + _value);
+        }
+    }
+
+    /**
+     * @notice Utility function that allows the user to transfer funds to their safe at
+     * the beginning of a deployment. When size is not specified, we fallback to using the
+     * maximum drip size on the network.
+     */
```

Comment:
So I'm OK with this, but you should know that it makes the UX of writing a script that you will execute on multiple networks a bit worse because you have to write logic to specify different values on all the networks. To be fair though, maybe it's reasonable to expect that of the user because the currencies and gas costs do vary significantly across networks.

Reply:
This only makes the UX worse if the user wants to send the maximum amount of tokens on each network, right?

Reply:
@RPate97 ^

Reply:
If the UX is only worse when the user is sending the max amount, I think it'd make sense to wait until we get some evidence that users want to send the max amount